<?xml version="1.0"?>
<launch>
    <arg name="record_path" default="$(find multi_localizer)/record/" />
	<arg name="roomba"      default="roomba1" />

	<!-- pose_recorder -->
	<node pkg="multi_localizer" type="pose_recorder" name="$(arg roomba)_pose_recorder" >
		<param name="ROBOT_NAME"       value="$(arg roomba)" />
		<param name="RECORD_FILE_PATH" value="$(arg record_path)" />
	</node>

</launch>